cardano_node_tests/cluster_scripts/conway_fast/run-cardano-smash

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -uo pipefail
+
+SOCKET_PATH="$(readlink -m "$CARDANO_NODE_SOCKET_PATH")"
+STATE_CLUSTER="${SOCKET_PATH%/*}"
+STATE_CLUSTER_NAME="${STATE_CLUSTER##*/}"
+
+export PGPASSFILE="$STATE_CLUSTER/pgpass"
```

Comment:
Is this needed here? It is already exported for db-sync, and as db-sync is required for smash, it will be available.

Reply:
It is needed here after all.